src/avram/migrator/runner.cr

```diff
@@ -172,10 +172,25 @@ class Avram::Migrator::Runner
 
   def ensure_migrated!
     if !pending_migrations.empty?
-      raise "There are pending migrations. Please run lucky db.migrate"
+      puts ""
+      puts error_background
+      puts pending_migrations_error.colorize.on_red.white
+      puts error_background
+      puts ""
```

Comment:
Should this just be a heredoc?

Reply:
I'm not sure... I'll have to see what `puts " ".colorize.on_red.white` does 🤔

Reply:
lol 😂  my editor keeps stripping out the blank space in the heredoc. But it does work ok if I do something like this:

```crystal
def blank_space
  " " * 50
end

puts <<-MESSAGE.colorize.on_red.white
#{blank_space}
 WARNING THERE IS A THING GOING ON HERE
 #{blank_space}
MESSAGE
```

What are your thoughts? I'm indifferent on this.

Reply:
If your editor is doing it, it's probably something to do with crystal fmt -- that tool annihilates whitespace wherever it can. I'm just nitpicking, I don't think it matters! 😆 Crystal tool fmt and I don't get along most of the time...